bindgen/src/bindings/cpp/templates/wrapper.hpp

```diff
@@ -14,6 +14,25 @@
 #include <type_traits>
 #include <variant>
 
+{%- for typ in ci.iter_types() %}
```

Comment:
This looks like it should be in Rust code somewhere. Is the order of custom type imports important? From my experience with C# generator `add_include` sorts the includes, so adding this here "before" the include statements would not influence the order.

Reply:
i totally forgot that you can also do things outside of templates :D, moved the import code to the wrapper generator constructor